src/GitVersionCore/GitVersionCore.csproj

```diff
@@ -47,6 +47,7 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Web" />
```

Comment:
What is needed from this reference. Am worried this could cause issues porting to .net core/mono which is happening in v4